operator/Makefile.core.mk

```diff
@@ -1,148 +0,0 @@
-# Copyright 2019 Istio Authors
```

Comment:
Is it safe to remove this file at this point?

Reply:
i don't think all of these have been ported to the root makefile yet?

Reply:
I think they have. However, for the sake for easier review (and as this PR requires quite a bit of approvals, apparently), I'm moving the deletion to a separate PR. It's not required to fix the build, anyway.